Ricochet/Assets/_Scripts/Managers/GameManager.cs

```diff
@@ -56,11 +99,55 @@ void Awake()
         {
             Destroy(gameObject);
         }
+        currentMatchTime = gameMatchTime;
+        if (playerSelectedData != null)
+        {
+            for (int i = 0; i < players.Length; i++)
```

Comment:
So are you saying loop through the number of inputs from rewired instead of our local storage of the players? If that's what you are saying then I agree but don't really know Rewired that well and I don't think we are even using it in the game manager currently?

Reply:
You're right. This will probably happen over this sprint

Reply:
You're right. This will probably happen over this sprint